modules/proxy-certificates.adoc

```diff
@@ -0,0 +1,75 @@
+
+// Module included in the following assemblies:
+//
+// * authentication/understanding-certificates.adoc
+
+[id="proxy-certificates_{context}"]
+= Proxy certificates
+
+== Purpose
+
+The `trustedCA` field of `proxy` is a reference to a ConfigMap that contains a
+user-provided trusted certificate authority (CA) bundle. This bundle is merged
+with the Red Hat Enterprise Linux CoreOS (RHCOS) trust bundle and injected into
+the trust store of platform components that make egress HTTPS calls. For
+example, `image-registry-operator` calls an external image registry to download
+images. If `trustedCA` is not specified, only the RHCOS trust bundle is used for
+proxied HTTPS connections.
+
+// Managing the RHCOS trust bundle is outside the scope of proxy. I need to contact that team for those details.
+
+The `trustedCA` field should only be consumed by a proxy validator. The
+validator is responsible for reading the certificate bundle from required key
+`ca-bundle.crt` and copying it to a ConfigMap named `trusted-ca-bundle` in the
+`openshift-config-managed` namespace. The namespace for the ConfigMap referenced
+by `trustedCA` is openshift-config`:
+
+[source,yaml]
+----
+apiVersion: v1
+	kind: ConfigMap
+ metadata:
+	 name: user-ca-bundle
+	 namespace: openshift-config
+	 data:
+	   ca-bundle.crt: |
+	     -----BEGIN CERTIFICATE-----
+	     Custom CA certificate bundle.
+	     -----END CERTIFICATE-----
+----
+
+== File path
+
+The user-provided trust bundle is represented as a ConfigMap. The ConfigMap is
+mounted into the filesystem of platform components that make egress HTTPS calls.
+Typically, Operators mount the ConfigMap to
+`/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`, but this not required by
+proxy and is at the Operator's discretion.
+
+== Expiration
+
+The user sets the expiration term of the user-provided trust bundle.
+
+// Contact the rhcos team for the default expiration term of the RHCOS system trust bundle.
```

Comment:
@miabbott Are you the correct content to help me build out these details for the customer?

Reply:
See https://github.com/openshift/openshift-docs/pull/18254#discussion_r356654866  for details about CA expiration.

TL;DR - If the user is dropping CAs onto the node via MachineConfigs, the expiration is defined in the certificate itself.  I don't know of any mechanism to control cert expiration of the trust bundle on the node.